cmd/eib/main.go

```diff
@@ -62,8 +74,19 @@ func handleImageConfig(configFile string) (*config.ImageConfig, error) {
 	return imageConfig, nil
 }
 
+func validateImageConfigDir(configDir string) error {
+	if configDir == "" {
+		return errors.New(fmt.Sprintf("-%s must be specified", argConfigDir))
```

Comment:
In general, you'd do:

```suggestion
		return fmt.Errorf("-%s must be specified", argConfigDir)
```

However, avoiding "dynamic" errors is almost always better. I'd just hardcode the name of the arg.